### DISCUSSION TITLE
c7n Discussions

### DISCUSSION THREAD

**Discussion by JohnHillegass** [cloud-custodian/cloud-custodian#7398] [General]
Is there any interest in enabling discussions on this repo now that it is in public beta? It seems like a really interesting way to throw around ideas, do Q+A, etc. The ability to upvote things seems useful too as an indicator of interest in features.

https://docs.github.com/en/discussions

**Comment by kapilt** [comment]
i'm game to explore.. i wasn't sure about entrenching github in this way, and have explored discourse (@ c7n.dev) as an alternative. but i'm coming around to it as inevitable.

**Comment by JohnHillegass** [comment]
Discourse seems like a good alternative to Gitter but for prioritizing feature requests etc, I think discussions are a great place. When enabled we would need to decide on categories of discussion

![image](https://user-images.githubusercontent.com/13325146/108604636-21cb2600-737d-11eb-9bf9-420d4231e11b.png)

They can be open-ended or Q/A-based. It may make the most sense to have an open-ended channel for provider-specific discussions and a catch-all Q/A for "help me" types of questions. I like the "help me" concept as there would be a good permalink and easy way for users to search for answers to common problems. 
![image](https://user-images.githubusercontent.com/13325146/108604644-2abbf780-737d-11eb-8dd5-a9be50823863.png)

**Comment by castrojo** [comment]
We've got sections and GitHub has the ability to migrate all our old issues so I've implemented this!
